converter/internal/common/common.go

```diff
@@ -32,3 +39,38 @@ func getValueOverrideHook() builder.ValueOverrideHook {
 		}
 	}
 }
+
+func GetUniqueLabel(label string, currentCount int) string {
+	if currentCount == 1 {
+		return label
+	}
+
+	return fmt.Sprintf("%s_%d", label, currentCount)
+}
+
+// prettyPrint attempts to pretty print the input slice. If prettyPrint fails,
+// the input slice is returned unmodified.
```

Comment:
Instead of saying `input slice` maybe add a note that it parses a river config.

Reply:
IMO and outside this scope but I think we should start typedef river configs from []byte to `type RiverConfig []byte`